js/main.js

```diff
@@ -4,36 +4,73 @@ const addBtn = document.querySelector('#addBtn');
 const title = document.querySelector('#title');
 const arthur = document.querySelector('#arthur');
 let bookObj = [];
+
+const navlist = document.querySelector('#listTab')
+const navaddnew = document.querySelector('#addNewTab')
+const navcontact = document.querySelector('#contactTab')
+const list = document.querySelector('#list')
+const addnew = document.querySelector('#addNew')
+const contact = document.querySelector('#contact')
+
+navlist.addEventListener('click', (e)=>{
+  e.preventDefault()
+  list.style.display = 'flex'
+  addnew.style.display = 'none'
+  contact.style.display = 'none'
+})
+navaddnew.addEventListener('click', (e)=>{
+  e.preventDefault()
+  list.style.display = 'none'
+  addnew.style.display = 'flex'
+  contact.style.display = 'none'
+})
+navcontact.addEventListener('click', (e)=>{
+  e.preventDefault()
+  list.style.display = 'none'
+  addnew.style.display = 'none'
+  contact.style.display = 'block'
+})
+window.addEventListener('load', (e)=>{
+  e.preventDefault()
+  list.style.display = 'flex'
+  addnew.style.display = 'none'
+  contact.style.display = 'none'
+})
+
 class Books {
-  /* eslint-disable */
-  constructor(bookTitle, arthur, id) {
+  constructor(bookTitle,arthur,id) {
     this.bookTitle = bookTitle,
     this.arthur = arthur,
-    this.id = id;
+    this.id = id
   }
 }
 class BookObj {
-  static loop() {
+ static  loop() {
     let objVar = '';
     for (let i = 0; i < bookObj.length; i += 1) {
       objVar += `<li id='lanzz'><p><span>${bookObj[i].bookTitle}</span> by <span>${bookObj[i].arthur}</span> </p> <button onclick = 'BookObj.removeBook(this.id)' id=${bookObj[i].id.toString()}>Remove</button></li>`;
     }
     bookList.innerHTML = objVar;
   }
 
-  static removeBook(id) {
-    filteredArr = bookObj.filter((item) => item.id !== id);
-    bookObj = filteredArr;
-    BookObj.loop();
-    localStorage.setItem('inputArr', JSON.stringify(bookObj));
-  }
-}
+  static  removeBook(id) {
+     filteredArr = bookObj.filter((item) => item.id !== id);
+     bookObj = filteredArr;
+     BookObj.loop();
+     localStorage.setItem('inputArr', JSON.stringify(bookObj));
+   }
 
+   static resetfields(){
+    document.querySelector('#title').value = ''
+    document.querySelector('#arthur').value = ''
+   }
+}
+ 
 addBtn.addEventListener('click', () => {
   if (title.value) {
     const titleValue = title.value;
     const arthurValue = arthur.value;
-    const newBook = new Books(titleValue, arthurValue);
+    const newBook = new Books(titleValue,arthurValue)
     bookObj.push(newBook);
     let objVar = '';
     for (let i = 0; i < bookObj.length; i += 1) {
```

Comment:
- [ ]  [OPTIONAL] In order to erase the form values after the submission you can do the following  

![image](https://user-images.githubusercontent.com/95243996/219022465-4f63d607-e042-40c4-af0b-27a29d3e6efa.png)